pkgs/development/libraries/librsb/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib, stdenv, fetchurl
+, gfortran
+, pkg-config, libtool
+, m4, gnum4
+, file
+# Memory Hierarchy (End-user can provide this.)
+, memHierarchy ? ""
+# Headers/Libraries
+, blas, zlib
+# RPC headers (rpc/xdr.h)
+, openmpi, ntirpc
+, help2man
+, doxygen
+, octave
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librsb";
+  version = "1.2.0.9";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "1ynrsgnvv1jfm8dv3jwjrip9x9icxv7w3qrk149025j6fbaza8gl";
+  };
+
+  # The default configure flags are still present when building
+  # --disable-static --disable-dependency-tracking
+  # Along with the --prefix=... flag (but we want that one).
+  configureFlags = [
+    "--enable-static"
+    "--enable-doc-build"
+    "--enable-octave-testing"
+    "--enable-sparse-blas-interface"
+    "--enable-fortran-module-install"
+    "--enable-pkg-config-install"
+    "--enable-matrix-types=all"
+  ]
+  ++ lib.optionals (zlib != null) [ "--with-zlib=${zlib}/lib/libz.so" ]
+  ++ lib.optionals (memHierarchy != "") [ "--with-memhinfo=${memHierarchy}" ];
+
+  # Ensure C/Fortran code is position-independent.
+  CFLAGS = [ "-fPIC" "-Ofast" ];
```

Comment:
```suggestion
  NIX_CFLAGS_COMPILE = [ "-fPIC" "-Ofast" ];
```

Reply:
How about the `FCFLAGS` for the fortran portion of the code? Does Nix have a `NIX_FCFLAGS_COMPILE`?

Reply:
I didn't find one so I don't think so.